src/components/Calculator.js

```diff
@@ -1,14 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Button from './Button';
 import './calculator.css';
+import caclulate from '../logic/calculate';
 
-const btnKeys = ['AC', '+/-', '%', '÷', '7', '8', '9', '×', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];
+const btnKeys = ['AC', '+/-', '%', '÷', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];
 
 function Calculator() {
+  const [state, setState] = useState({ total: null, next: null, operation: null });
+
+  const { total, next, operation } = state;
+
+  const handleClick = (e) => {
+    const btnValue = e.target.innerHTML;
+    setState(caclulate(state, btnValue));
+  };
```

Comment:
- [ ] As stated in the review summary template earlier, the second operand is not yet being displayed on the screen. For example if I calculate `2 + 5`, only figure 2 was displayed to the screen; kindly work on it. 👍

Reply:
noted, thanks for that review